nexis/lib/pages/auth/register_page.dart

```diff
@@ -0,0 +1,361 @@
+                  shape: RoundedRectangleBorder(
+                    borderRadius: BorderRadius.circular(5),
+                  ),
+                  child: Padding(
+                    padding: const EdgeInsets.all(20.0),
+                    child: Column(
+                      mainAxisSize: MainAxisSize.min,
+                      children: [ 
+                        const CustomColumn(
+                          ColumnType.type1,
+                          largeLabel: 'Create an account',
+                          mediumLabel: 'Email',
+                          /*
+                            Add here for further implementation
+                          */ 
+                        ),
+                        const SizedBox(height: 20),
+                        const CustomColumn(
+                          ColumnType.type3,
+                          mediumLabel: 'Username',
+                          /*
+                            Add here for further implementation
+                          */ 
+                        ),
+                        const SizedBox(height: 20),
+                        const CustomColumn(
+                          ColumnType.type3,
+                          mediumLabel: 'Password',
+                          obscureText: true,
+                          /*
+                            Add here for further implementation
+                          */ 
+                        ),
+                        const SizedBox(height: 20),
+                        const CustomColumn(
+                          ColumnType.type3,
+                          mediumLabel: 'Repeat Password',
+                          obscureText: true,
+                          /*
+                            Add here for further implementation
+                          */ 
+                        ),
+                        const SizedBox(height: 20),
+                        CustomColumn(
+                          ColumnType.type3,
+                          mediumLabel: 'Date of birth',
+                          hint: 'Month-Day-Year',
+                          controller: dateController,
+                          onTap: selectDate,
+                          readOnly: true,
+                        ),
+                        const SizedBox(height: 20),
+                        CustomColumn(
+                          ColumnType.type4,
+                          buttonText: 'Register',
+                          //onPressed: {Function},
+                          mediumBody: 'Already have an account?',
+                          recognizer: TapGestureRecognizer()..onTap = RouteChange(context, RouteNames.authPage).reDir,
+                        ),
+                      ],
+                    ),
+                  ),
+                ),
+              ]
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget RegisterPageTablet(BuildContext context) {
```

Comment:
this is a exact copy of the RegisterPageDesktop, so remove this one and only use RegisterPageDesktop